src/features/selectedPost.ts

```diff
@@ -0,0 +1,27 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { Post } from '../types/Post';
+
+export interface SelectedPostState {
+  selectPost: Post | null;
+}
+
+const initialState: SelectedPostState = {
+  selectPost: null,
```

Comment:
hm, seems you misunderstood my comment :slightly_smiling_face: 

name of this property was correct (`selectedPost`). So please, revert it - rename it back to `selectedPost`

I meant __function__, i.e. action name (see next comment)